libqtile/backend/wayland/drawer.py

```diff
@@ -90,8 +90,11 @@ def draw(
 
         # Paint RecordingSurface operations our window's ImageSurface
         with cairocffi.Context(self._source) as context:
+            context.save()
+            context.set_operator(cairocffi.OPERATOR_SOURCE)
```

Comment:
By using `OPERATOR_SOURCE` in the `drawer.draw` method, we overwrite any existing contents in the drawer. This should stop the overlay/alpha blending issues seen in the screenshot above.

I'm not aware of any instance when we actually want `drawer.draw` to overlay previously rendered contents.